public/usage-examples/graphics/draw_circle_on_bitmap/draw_circle_on_bitmap-1-red-planet-top-level.cs

```diff
@@ -0,0 +1,27 @@
+using static SplashKitSDK.SplashKit;
+
+// Create a bitmap to draw on
+Bitmap planet = CreateBitmap("planet", 400, 400);
+
+// Fill background with dark color
+ClearBitmap(planet, ColorBlack());
+
+// Create color
+Color red = ColorRed();
+
+// Draw the main planet circle
+FillCircleOnBitmap(planet, RGBAColor(180, 0, 0, 255), 200, 200, 150);
+DrawCircleOnBitmap(planet, red, 200, 200, 150);
+
+// Add some surface details with smaller circles
+for(int i = 0; i < 15; i++)
+{
+    double x = Rnd(100, 300)  // Random between 100 and 300
+    double y = Rnd(100, 300)   // Random between 100 and 300
+    double size = Rnd(10, 30)  // Random between 10 and 30
+    DrawCircleOnBitmap(planet, red, x, y, size);
+}
+
+// Save and free the bitmap
+SaveBitmap(planet, "red_planet");
+FreeBitmap(planet);
```

Comment:
This code is also filled with errors, in calling the colours, missing semi colons. I will leave this one for you to fix and correct.

Also, on line 26, please ensure the name of the saved file matches the name you used in the other programs.